corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1161,10 +1161,6 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
-        onBeforeDetach: function () {
-            CaseTileListView.__super__.onBeforeDetach.apply(this, arguments);
```

Comment:
Is detaching the small screen listener not important?

Reply:
`CaseTileListView` extends `CaseListView` so `onBeforeDetach` defined in `CaseListView` will still apply

Reply:
makes sense